Formula/carthage.rb

```diff
@@ -13,6 +13,7 @@ class Carthage < Formula
     sha256 "9eb176f9fcbd997ae7d692362d526cafedd87bc506de92db7db2f6dd9bdf9775" => :sierra
   end
 
+  depends_on :macos => :el_capitan
```

Comment:
While it may run on 10.11, there will only be bottles for 10.12 and 10.13 due to the Xcode 9 requirement, and a source build won't be possible unless you upgrade to 10.12, so this is redundant with the Xcode 9 requirement below.